rllib/evaluation/worker_set.py

```diff
@@ -231,6 +244,204 @@ def sync_weights(
         elif self.local_worker() is not None and global_vars is not None:
             self.local_worker().set_global_vars(global_vars)
 
+    def add_policy(
```

Comment:
New API for WorkerSet:
- add_policy(): Similar to `Algorithm.add_policy()`.
- WorkerSet.add_policy_to_workers(): New static helper utility for adding a new policy (by instance or options) to a list of (local and/or remote) workers.